Check if allcfgconv supports "-c", fallback to decode_passwords (by PeterPawn) if not.

--- root/usr/lib/callmonitor/modules/config.sh
+++ root/usr/lib/callmonitor/modules/config.sh
@@ -1,6 +1,12 @@
 #! /bin/ash
 cfg_cat() {
+if allcfgconv -? 2>&1 | grep -q " -c "; then
 allcfgconv -C "$1" -e -c -o -
+elif type -p decode_passwords >/dev/null 2>&1; then
+allcfgconv -C "$1" -e    -o - | decode_passwords
+else
+echo "Error: no password decryption method available" >&2
+fi
 }
 cfg_parse() {
 sed -ne '
